### PR TITLE
Adding EasyAnalyzer to LucyJS

### PR DESCRIPTION
**Pull request by QuantamHD**
Adding the EasyAnalyzer to Lucy gets us one step closer to getting the BeyondBasicTutorial working in Javascript.

### CODE REVIEW

File: test/lucy.test.js [hunk 1]
```diff
@@ -24,4 +24,18 @@ describe("lucy", () => {
...
+  it("has a EasyAnalyzer method", () => {
+    expect(lucy.EasyAnalyzer).toBeDefined();
+  });
+
+  it("can create a EasyAnalyzer with new", () => {
+    const easy_analyzer = new lucy.EasyAnalyzer("en");
+    expect(easy_analyzer).toBeDefined();
+  });
+
+  it("can create a EasyAnalyzer without new", () => {
```
Comment (Flaque) [inline test/lucy.test.js:37]: Why is this repeated?

Comment (Flaque) [reply test/lucy.test.js:37]: Ah nvm

**Comment by Flaque** [comment]
LGTM

**Comment by QuantamHD** [comment]
Thanks FAM

**Pull request merged**